php/Terminus/Workflow.php

```diff
@@ -123,4 +122,25 @@ public function status($key=null) {
      return $this->status;
    }
 
+   public function isFinished() {
+     if ($this->status->result) {
+       return true;
+     } else {
+       return false;
+     }
+   }
+
+   public function isSuccessful() {
+     if ($this->isFinished() && $this->status->result == 'succeeded') {
```

Comment:
holy logic, you are so right. I was trying to figure out how PHP coerces stuff and gave up.